frontend/packages/console-app/src/components/quick-starts/QuickStartController.tsx

```diff
@@ -5,7 +5,7 @@ import QuickStartFooter from './controller/QuickStartFooter';
 import { QuickStartContext, QuickStartContextValues } from './utils/quick-start-context';
 
 type QuickStartControllerProps = {
-  quickStart: QuickStart;
+  quickStart: QuickStart & { nextQuickStarts?: QuickStart[] };
```

Comment:
Why not a second `nextQuickStarts` prop here? Any benefit of reusing a QuickStart object which is now a common QuickStart object?

```suggestion
  quickStart: QuickStart;
  nextQuickStarts?: QuickStart[];
```